frontend/src/views/producer/dashboard/reservation-list/index.js

```diff
@@ -0,0 +1,172 @@
+  }
+  & > button {
+    position: absolute;
+    right: 1rem;
+    top: 1rem;
+  }
+`;
+const ScrollContainer = styled.div`
+  margin: 3rem;
+  max-width: 1600px;
+  width: auto;
+  & > ul {
+    margin-top: 2rem;
+  }
+`;
+
+const LiStyled = styled.li`
+  list-style: none;
+  margin: 1rem;
+  display: flex;
+
+  & > div {
+    display: inline-block;
+    margin-left: 1rem;
+
+    & > ul {
+      margin: 0.5rem;
+      padding-left: 1rem;
+    }
+  }
+`;
+
+const ReservationListItem = ({ reservation, event }) => {
+  const {
+    customerType,
+    name,
+    schoolName,
+    class: className,
+    phone,
+    email,
+    paymentCompleted,
+    tickets,
+  } = reservation;
+  const isPrivateCustomer = customerType === 'private';
+  const totalCost = tickets.reduce((acc, curr) => {
+    const { price } = event.catalogById(curr.priceId);
+    return (acc += price * curr.noOfTickets);
+  }, 0);
+
+  return (
+    <LiStyled key={reservation.id}>
+      <CheckBox checked={paymentCompleted} />
+      <div>
+        {isPrivateCustomer ? (
+          <Typography
+            style={{ margin: 0, fontSize: '1.5rem' }}
+            type="largebody"
+          >
+            {name}
+          </Typography>
+        ) : (
+          <Typography
+            style={{ margin: 0, fontSize: '1.5rem' }}
+            type="largebody"
+          >
+            {name}, {schoolName}, {className}
+          </Typography>
+        )}
+        <br />
+        <Typography type="body">
+          {phone} {email && '· ' + email}
+        </Typography>
+        <ul>
+          {tickets.map(ticket => {
+            const catalog = event.catalogById(ticket.priceId);
+            return (
+              <li key={ticket.id}>
+                <Typography type="body">
+                  {catalog.ticketDescription} {ticket.noOfTickets} kpl{' '}
+                  {catalog.price * ticket.noOfTickets} €
+                </Typography>
+              </li>
+            );
+          })}
+        </ul>
+        <Typography
+          type="largebody"
+          color={event.themeColor}
+          style={{ fontSize: '1.2rem' }}
+        >
+          Yhteensa: {totalCost} €
+        </Typography>
+      </div>
+    </LiStyled>
+  );
+};
+
+class ReservationList extends React.Component {
+  render() {
+    const { palette } = this.props.theme;
+    const { reservationsAndOrders, selectedEvent, ui } = this.props.store;
+    if (!reservationsAndOrders || !selectedEvent) return null;
+    const reservations = values(reservationsAndOrders).filter(
+      r => r.eventId.id === selectedEvent.id,
+    );
+    console.log(reservations);
+    if (!reservations) return null;
+
+    return createPortal(
+      <FullScreenModal shown={ui.orderAndPayment.listingShown}>
+        <Button
+          icon="close"
+          backgroundColor={palette.red}
+          onClick={ui.orderAndPayment.toggleShowListing}
+        >
+          Takaisin
+        </Button>
+        <ScrollContainer>
+          <p>
```

Comment:
Maybe would be better to use div instead of p here, since what is inside is not a paragraph of text.